driver/src/orderbook/streamed/error.rs

```diff
@@ -0,0 +1,13 @@
+use thiserror::Error;
+
+#[derive(Clone, Copy, Debug, Error)]
+pub enum Error {
+    #[error("unknown token")]
+    UnknownToken,
+    #[error("unknown order")]
+    UnknownOrder,
+    #[error("order already exists")]
+    OrderAlreadyExists,
+    #[error("math underflow")]
```

Comment:
You mentioned that the debug information around which order/token is causing troubles was in another PR? Up to you if you want to add it here.